pkg/apis/serving/v1alpha1/revision_validation_test.go

```diff
@@ -700,7 +713,10 @@ func TestRevisionValidation(t *testing.T) {
 				ConcurrencyModel: "Multi",
 			},
 		},
-		want: &apis.FieldError{Message: "Invalid resource name: length must be no more than 63 characters", Paths: []string{"metadata.name"}},
+		want: &apis.FieldError{
+			Message: "not a DNS 1035 label: [must be no more than 63 characters]",
+			Paths:   []string{"metadata.name"},
+		},
 	}}
```

Comment:
do you want a test with both error conditions?
Also you can use validation.go:MaxLenError to generate check string.

Reply:
FWIW, you can collapse 2 tests into 1 :-)

Reply:
Done for Revision.